revolv/static/css/screen.css

```diff
@@ -11005,3 +11579,25 @@ label.checkmark.animate:after {
     width: 100%;
   }
 }
+
+
+/* 30-Jun */
```

Comment:
Same thing with this comment. Please update it to describe what the styles below apply to.